packages/kit/src/exports/node/index.js

```diff
@@ -58,8 +61,10 @@ function get_raw_body(req, body_size_limit) {
 
 				size += chunk.length;
 				if (size > length) {
-					req.destroy(
-						new Error(
+					cancelled = true;
+					controller.error(
```

Comment:
I'm not well-versed with the details of this code section - could you explain briefly why this change is necessary?

Reply:
This returns an error in the RequestHandler and from there the status code gets into the Response, otherwise the connection is simply terminated without a response. `cancelled = true` is needed so that `controller.close()` does not occur in the `req.on('end', () => ...)`  because calling `close()` on a closed controller causes an unhandled exception.